README.md

```diff
@@ -348,6 +348,11 @@ When building on MacOS M1, if the build fails while trying to build OpenSSL or W
 To build on a 32-bit Raspbian GNU/Linux 11 on 64-bit hardware, the OpenSSL library must be manually configured. This is due to the OpenSSL autoconfiguration script detecting 64-bit hardware and emitting 64-bit ARM assembly instructions which are not allowed in 32-bit executables. A 32-bit ARM version of OpenSSL can be configured by setting 32-bit ARM platform:
 `cmake .. -DBUILD_OPENSSL_PLATFORM=linux-armv4`
 
+### Threadpool for Signaling Channel messages
+The threadpool is enabled by default, and starts with 3 threads that it can increase up to 5 if all 3 are actively in use. To change these values to better match the resources of your use case
```

Comment:
Do we want to make this configurable instead? Is it expected customers would not require more than 5?

Reply:
That's why I've pointed out the location of the defines for customers to change in this README. I would have preferred to have placed them in a more common library but that would require a different constructor for slgnaling client that allows you to pass in a given threadpool. Possibly an alternative to explore later.

Reply:
Hmm, we can add the 2 parameters as part of SignalingClientInfo though. Having customers change the `#defs` and rebuild the SDK to be used as a dependency in another project gets messy. If we are allowing customers to configure this, having it as part of clientInfo is cleaner